src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudApiService.scala

```diff
@@ -0,0 +1,158 @@
+package org.broadinstitute.dsde.firecloud
+
+import akka.http.scaladsl.model.{ContentTypes, HttpHeader, StatusCodes}
+import akka.http.scaladsl.server
+import akka.stream.Materializer
+import org.broadinstitute.dsde.firecloud.model.{ModelSchema, UserInfo, WithAccessToken}
+import org.broadinstitute.dsde.firecloud.service._
+import org.broadinstitute.dsde.firecloud.utils.StandardUserInfoDirectives
+import org.broadinstitute.dsde.firecloud.webservice._
+import org.slf4j.LoggerFactory
+
+import scala.concurrent.ExecutionContext
+import scala.language.postfixOps
+
+trait FireCloudApiService extends CookieAuthedApiService
```

Comment:
I'm migrating this to work more like how rawls works. It's not done, and I still need to break out some of the monitors which are commented out below